vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/TouchSteps.java

```diff
@@ -92,6 +100,33 @@ public void typeTextInField(String text, Locator locator)
                 .ifPresent(e -> keyboardActions.typeText(e, text));
     }
 
+    /**
+     * Swipes to element in <b>direction</b> direction with duration <b>duration</b>
+     * @param direction direction to swipe, either <b>UP</b> or <b>DOWN</b>
+     * @param locator locator to find an element
+     * @param duration between a pointer moves from the start to the end of the swipe coordinates
+     */
+    @When("I swipe $direction to element located `$locator` with duration $swipeDuration")
+    public void swipeToElement(SwipeDirection direction, Locator locator, Duration duration)
+    {
+        locator.getSearchParameters()
+               .setVisibility(Visibility.VISIBLE)
```

Comment:
it's so by default

Reply:
yes, but we can also set visibility by using locators, shouldn't we prevent user from like accident usage of visibility modifiers and ensure that the state of element should be visible?

Reply:
nope, if you want prevent it, then validate and throw an error, but you shouldn't change input parameters silently